iac-extensions/arm/src/main/resources/org/sonar/l10n/azureresourcemanager/rules/azureresourcemanager/S6383.html

```diff
@@ -0,0 +1,105 @@
+<p>Disabling Role-Based Access Control (RBAC) on Azure resources can reduce an organization’s ability to protect itself against access controls being
+compromised.</p>
+<p>To be considered safe, access controls must follow the principle of least privilege and correctly segregate duties amongst users. RBAC helps
+enforce these practices by adapting the organization’s access control needs into explicit role-based policies: It helps keeping access controls
+maintainable and sustainable.</p>
+<p>Furthermore, RBAC allows operations teams to work faster during a security incident. It helps to mitigate account theft or intrusions by quickly
+shutting down accesses.</p>
+<h2>Ask Yourself Whether</h2>
+<ul>
+  <li> This Azure resource is essential for the information system infrastructure. </li>
+  <li> This Azure resource is essential for mission-critical functions. </li>
+  <li> Compliance policies require access to this resource to be enforced through the use of Role-Based Access Control. </li>
+</ul>
+<p>There is a risk if you answered yes to any of those questions.</p>
+<h2>Recommended Secure Coding Practices</h2>
+<ul>
+  <li> Enable Azure RBAC when the Azure resource supports it. </li>
+  <li> For Kubernetes clusters, enable Azure RBAC if Azure AD integration is supported. Otherwise, use the built-in Kubernetes RBAC. </li>
+</ul>
+<h2>Sensitive Code Example</h2>
+<p>For <a
+href="https://learn.microsoft.com/en-us/azure/templates/microsoft.containerservice/2023-03-01/managedclusters?pivots=deployment-language-arm-template">AKS
+Azure Kubernetes Service</a>:</p>
+<pre data-diff-id="1" data-diff-type="noncompliant">
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "type": "Microsoft.ContainerService/managedClusters",
+      "apiVersion": "2023-03-01",
+      "properties": {
+        "aadProfile": {
```

Comment:
At first, I thought this is a type but it seems to be the acronym for `Azure Active Directory` :smile: